code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1214,11 +1214,43 @@
 	else
 		to_chat(usr, "<span class='info'>You moved while counting. Try again.</span>")
 
-/mob/living/carbon/human/proc/set_species(var/new_species_name, var/force_organs, var/default_colour)
+//Record_organ datum, stores as much info as it can about each organ
+/datum/record_organ
+	var/name
+
+/datum/record_organ/external
+	var/brute_damage
+	var/burn_damage
+	var/list/wounds = list()
+	var/status_flags
+
+/datum/record_organ/internal
```

Comment:
what do these do?
planned for carrying over organ damage and implants and shrapnel and such?

Reply:
It was the previous implementation which reapplied damages on a per-limb basis (or at least the equivalent), I think it was too overengineered and these are just some of the traces left.

Reply:
I re-added them.